__tests__/integrations/Dialog/ApprovelDialog.test.ts

```diff
@@ -33,9 +33,9 @@ describe("Dialog consultation", () => {
 
     const consult = await request.get("/v1/dialog").query({ _id: dialogId });
 
-    expect(response.statusCode).toBe(204);
+    expect(response.status).toBe(204);
```

Comment:
does this status come from TS-jest?

Reply:
This status come directly from ```@types/supertest```. There's no definition for ```response.status```

Reply:
why use status and not statusCode? let me know the supertest uses statusCode

Reply:
See this exemple: https://tutorialedge.net/typescript/testing-typescript-api-with-jest/